### PR TITLE
Expanded Paper to support the content text

### PR DESCRIPTION
**Pull request by jcomack**
Now the text will also be managed by Paper.

### CODE REVIEW

File: js/values/Paper.js [hunk 1]
```diff
@@ -3,8 +3,9 @@
  * @param {string} keyword
  * @constructor
  */
-var Paper = function( keyword ) {
```
Comment (atimmer) [inline js/values/Paper.js]: `text` is missing JSDoc

**Comment by atimmer** [comment]
Please add tests

**Comment by atimmer** [comment]
CR Done 👷🏻

**Comment by terw-dan** [comment]
It currently breaks the analyzer queue

**Comment by terw-dan** [comment]
CR Done :+1:

**Comment by terw-dan** [comment]
Acceptance done :+1:

**Pull request merged**